### PR TITLE
Add QTime::isValid(...) static method

### PR DESCRIPTION
**Pull request by Montel**

### CODE REVIEW

File: crates/cxx-qt-lib/src/core/qtime.rs [hunk 1]
```diff
@@ -157,6 +161,9 @@ impl QTime {
     pub fn secs_to(&self, t: Self) -> i32 {
         ffi::qtime_secs_to(self, t)
     }
+    pub fn is_valid_time(h: i32, m: i32, s: i32, ms: i32) -> bool {
```
Comment (ahayzen-kdab) [inline crates/cxx-qt-lib/src/core/qtime.rs:164]: add space above

Comment (ahayzen-kdab) [reply crates/cxx-qt-lib/src/core/qtime.rs:164]: and add documentation line

File: crates/cxx-qt-lib/src/core/qtime.rs [hunk 2]
```diff
@@ -157,6 +161,12 @@ impl QTime {
     pub fn secs_to(&self, t: Self) -> i32 {
         ffi::qtime_secs_to(self, t)
     }
+
+    /// Returns true if the specified time is valid; otherwise returns false.
+    /// The time is valid if h is in the range 0 to 23, m and s are in the range 0 to 59, and ms is in the range 0 to 999.
+    pub fn is_valid_time(h: i32, m: i32, s: i32, ms: i32) -> bool {
```
Comment (ahayzen-kdab) [inline crates/cxx-qt-lib/src/core/qtime.rs:167]: why is this called `is_valid_time` and not `is_valid` ?

Comment (Montel) [reply crates/cxx-qt-lib/src/core/qtime.rs:167]: Because there is already a is_valid() method (without arguments)

Comment (ahayzen-kdab) [reply crates/cxx-qt-lib/src/core/qtime.rs:167]: Ah i see this is a static method, wonder if there is a common way of naming this in Rust, or if `_time` is enough

**Review by LeonMatthesKDAB — APPROVED** [review]
LGTM

**Pull request merged**